app/src/main/java/com/example/countryinfo/CountriesPreviewViewModel.kt

```diff
@@ -11,15 +11,11 @@ class CountriesPreviewViewModel(
     private val countryApi: ICountriesApi
 ) : ViewModel() {
 
-    init {
-        getCountries()
-    }
-
     private val countriesMutableLiveData = MutableLiveData<CountriesPreviewViewState>()
-    val pollLiveData: LiveData<CountriesPreviewViewState> = countriesMutableLiveData
+    val countriesLiveData: LiveData<CountriesPreviewViewState> = countriesMutableLiveData
     private lateinit var disposable: Disposable
 
-    private fun getCountries() {
+    fun getCountries() {
```

Comment:
No need to do so. Since ViewModel can be initialised lazily in tests the `init` block will be invoked on the first call on `ViewModel` reference, so you should be safe. Let's revert those changes.

Reply:
Changed.